styles/nav.scss

```diff
@@ -119,7 +119,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
-    .daptiv-nav-menuitem-alternate {
+    .daptiv-nav-menuitem.alternate {
```

Comment:
hmmmm.... that's unfortunate. Well, if we're going to leave it separate, we should at least move it up near the other rule.  Also this rule doesn't need to `@extend .daptiv-nav-menuitem-base`